boards/crazyflie/src/stm32-board.adb

```diff
@@ -1,6 +1,6 @@
 ------------------------------------------------------------------------------
 --                                                                          --
---                    Copyright (C) 2015, AdaCore                           --
+--                  Copyright (C) 2015, 2020 AdaCore                        --
```

Comment:
We use a dash `-` here between the two years: `2015-2020`.

Reply:
I’ve done this, though I was intending to indicate that 2015, 2020 were the years in which changes were made, rather than 2015-2020, which in GCC would AFAIK mean that changes were made in all 6 years.

Reply:
This is our policy, we also put the range.